examples/scroll/README.md

```diff
@@ -0,0 +1,28 @@
+# Scroll Elements Into View
+
+**Test Code**: [index.js](index.js)  
+
+TestCafe [actions](https://devexpress.github.io/testcafe/documentation/guides/basic-guides/interact-with-the-page.html) have a built-in scrolling mechanism. In rare cases you might need to scroll the page manually. This example demonstrates two common ways to do this: a `hover` action and a [ClientFunction](https://devexpress.github.io/testcafe/documentation/guides/basic-guides/obtain-client-side-info.html)
+
+The test page has 2 buttons. They are located 2000 px and 4000 px to the right respectively, both outside of viewport.
```

Comment:
```suggestion
The test page has 2 buttons. They are located 2000 px and 4000 px to the right, respectively (both outside of viewport).
```
Can a senior tech writer review this sentence?

Reply:
I've asked a senior, he suggests:

> They are located at 2000 px and 4000 px along the X-axis, respectively (both outside of viewport).

Reply:
> They are located at 2000 px and 4000 px along the X-axis, respectively (both outside of viewport).